### PR TITLE
Debugging

### PR DESCRIPTION
**Pull request by Armandoxh**
Fixed the issues the app should run when you npm start it now

### CODE REVIEW

File: controllers/auth.js [hunk 1]
```diff
@@ -1,56 +1,57 @@
...
+  } catch (error) {
+    return res.status(500).json({
+      status: 500,
+      message: "Something went wrong!",
+      requestAt: new Date().toLocaleString(),
+    });
+  }
+};
+
+const login = () => {};
```
Comment (rhine1217) [inline controllers/auth.js:50]: Randy wrote this 'login' part in his PR - I don't know if you can cancel this particular commit to avoid a conflict?

Comment (Armandoxh) [reply controllers/auth.js:50]: did that branch get merged?

Comment (rhine1217) [reply controllers/auth.js:50]: I just merged it... it should be in submaster branch now

Comment (rhine1217) [reply controllers/auth.js:50]: everything else looks good to go (if there's no merging conflicts to resolve)

**Pull request closed without merging**